src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -37,6 +37,7 @@
         >
         <small class="form-text">
           {{ 'Search for an existing customer by typing the first letters of his/her name.'|trans({}, 'Admin.Orderscustomers.Help') }}
+          {{ 'You can use " OR " for a multiple search.'|trans({}, 'Admin.Orderscustomers.Help') }}
```

Comment:
```suggestion
          {{ 'Use OR to broaden your search'|trans({}, 'Admin.Orderscustomers.Help') }}
```

If I understand well, if a user enters "John Doe", only the results containing both John AND Doe will appear. But the user can use OR if he/she wants to display results containing either John or Doe for example, John Sims and Nicolas Doe?